website/content/partials/api/restricted-endpoints.mdx

```diff
@@ -54,6 +55,7 @@ API path                                    | Root | Admin
 `sys/sealwrap/rewrap`                       | YES  | NO
 `sys/step-down`                             | YES  | NO
 `sys/storage`                               | YES  | NO
+`sys/sync/*                                 | YES  | YES
```

Comment:
We should also update /api-docs/system/secrets-sync. The restricted endpoint warning is currently only under the "config" endpoint. If the entire subset of endpoints under `sys/sync` are now restricted, we should move the alert to the top of the page

Reply:
Oops! They haven't changed, I wasn't thinking very clearly. It's just the config endpoint and it is reachable on both now.